diff --git a/src/shopware/components/override/VPAlgoliaSearchBox.vue b/src/shopware/components/override/VPAlgoliaSearchBox.vue
index ce2302b405f5be71aa7e0471f84bb2d37ecf5cfa..5edcd29172cf85a92bf1759d115b131216e6c47a 100644
--- a/src/shopware/components/override/VPAlgoliaSearchBox.vue
+++ b/src/shopware/components/override/VPAlgoliaSearchBox.vue
@@ -73,6 +73,7 @@ function initialize(userOptions: DefaultTheme.AlgoliaSearchOptions) {
             })
         },
 
+        /*
         // @ts-expect-error vue-tsc thinks this should return Vue JSX but it returns the required React one
         hitComponent({ hit, children }) {
             return {
@@ -83,7 +84,7 @@ function initialize(userOptions: DefaultTheme.AlgoliaSearchOptions) {
                 key: undefined,
                 props: { href: hit.url, children }
             }
-        },
+        },*/
 
         // https://www.algolia.com/doc/api-reference/search-api-parameters/
         searchParameters: {
diff --git a/src/shopware/composables/Sidebar.ts b/src/shopware/composables/Sidebar.ts
index d785e2702f19ab18b6950ebd902b649cc31f2051..9ea43a9b0718342645d9f15b8bfc416777ad59f3 100644
--- a/src/shopware/composables/Sidebar.ts
+++ b/src/shopware/composables/Sidebar.ts
@@ -491,7 +491,7 @@ export function transformLinkToSidebar(root: string, link: string, ignore: strin
                         const items = metas['index.md']?.items || [];
                         index = {
                             link: surroundWithSlash(as),
-                            text: getTitleFromFilename(as),
+                            text: metas[file]?.title || getTitleFromFilename(as),
                             items: items,
                             collapsed: getCollapsed(0, items),
                             meta: metas['index.md']?.meta || {},
@@ -501,7 +501,7 @@ export function transformLinkToSidebar(root: string, link: string, ignore: strin
                         // special handle root links
                         inIndex.push({
                             link: `${surroundWithSlash(as)}${removeExtension(file)}.html`,
-                            text: metas[file].title || getTitleFromFilename(file),
+                            text: metas[file]?.title || getTitleFromFilename(file),
                             items: [],
                             collapsed: getCollapsed(0, []),
                             meta: metas[file]?.meta || {},
@@ -522,7 +522,7 @@ export function transformLinkToSidebar(root: string, link: string, ignore: strin
                 if (links.length || hasIndex) {
                     reduced.push(nullifyLink({
                         link: hasIndex ? `${surroundWithSlash(as)}${file}/` : '#',
-                        text: getTitleFromFilename(file),
+                        text: metas[file]?.title || getTitleFromFilename(file),
                         // @ts-ignore
                         items: links,
                         collapsed: getCollapsed(0, links),
